hibernate-ogm/nosql-with-hibernate-ogm-101/hibernate-ogm-demo-nosql-with-hibernate-ogm-101-part-3/src/main/java/org/hibernate/ogm/demos/ogm101/part3/repo/PersonRepository.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.demos.ogm101.part3.repo;
+
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+
+import org.hibernate.ogm.demos.ogm101.part3.model.Hike;
+import org.hibernate.ogm.demos.ogm101.part3.model.Person;
+
+/**
+ * @author Gunnar Morling
+ */
+@ApplicationScoped
+public class PersonRepository {
```

Comment:
Same here: It is pretty obvious but I would add a comment to explain what this class does